TerraformCLI/task.json

```
@@ -1,151 +1,150 @@
 {
-  "id": "721c3f90-d938-11e8-9d92-09d7594721b5",
-  "name": "TerraformCLI",
-  "friendlyName": "Terraform CLI",
-  "description": "Execute terraform cli commands",
-  "author": "Charles Zipp",
-  "helpMarkDown": "",
-  "category": "Utility",
-  "visibility": [
-    "Build",
-    "Release"
-  ],
-  "demands": [],
-  "version": {
-    "Major": "0",
-    "Minor": "2",
-    "Patch": "4"
-  },
-  "minimumAgentVersion": "1.95.0",
-  "instanceNameFormat": "terraform $(command)",
-  "groups" : [
-    {
-      "name": "backendAzureRm",
-      "displayName": "AzureRM Backend Configuration",
-      "isExpanded": false,
-      "visibleRule": "command = init && backendType = azurerm"
+    "id": "721c3f90-d938-11e8-9d92-09d7594721b5",
+    "name": "TerraformCLI",
+    "friendlyName": "Terraform CLI",
+    "description": "Execute terraform cli commands",
+    "author": "Charles Zipp",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "demands": [],
+    "version": {
+        "Major": "0",
+        "Minor": "2",
+        "Patch": "4"
     },
-    {
-      "name": "variables",
-      "displayName": "Variables",
-      "isExpanded": false,
-      "visibleRule": "command = validate || command = plan || command = apply"
+    "minimumAgentVersion": "1.95.0",
+    "instanceNameFormat": "terraform $(command)",
+    "groups": [{
+            "name": "backendAzureRm",
+            "displayName": "AzureRM Backend Configuration",
+            "isExpanded": false,
+            "visibleRule": "command = init && backendType = azurerm"
+        },
+        {
+            "name": "variables",
+            "displayName": "Variables",
+            "isExpanded": false,
+            "visibleRule": "command = validate || command = plan || command = apply"
```

@jakebair add destroy here otherwise users will not be able to provide vars to the destroy command
```suggestion
            "visibleRule": "command = validate || command = plan || command = apply || command = destroy"
```

---

I forgot to check-in that file, I had modified the id (so that I could have both the published version and my working version in my VSTS account) but forgot about it